### PR TITLE
Fix non image cards

### PR DESCRIPTION
**Pull request by Cestrus**
Fixed no poster situation
Some chahges styles
Tested

### CODE REVIEW

**Comment by Cestrus** [comment]
Снимок

File: src/components/FilmPage/FilmPage.jsx [hunk 1]
```diff
@@ -16,10 +15,6 @@ function FilmPage(props) {
     );
 }
 
-// function FilmInfo() {
-//     return <div>Film`&apos;`s info component!</div>;
-// }
-
 FilmPage.propTypes = {
     match: PropTypes.object,
```
Comment (LordGemma) [inline src/components/FilmPage/FilmPage.jsx]: Could you write more detailed prop types for `match`. For example `match: PropTypes.shape({....})`

**Comment by LordGemma** [comment]
Please update your MR

File: src/components/FilmPage/FilmPage.jsx (hunk 1 above)
Comment (Cestrus) [reply src/components/FilmPage/FilmPage.jsx]: fixed PropTypes, change style in MainContainer (delete padding-top becouse appear scroll in NotFound

**Pull request merged**